share/p2p/discovery/metrics.go

```diff
@@ -130,7 +121,7 @@ func initMetrics(d *Discovery) (*metrics, error) {
 	return metrics, nil
 }
 
-func (m *metrics) observeFindPeers(ctx context.Context, canceled, isEnoughPeers bool) {
+func (m *metrics) observeFindPeers(ctx context.Context, isEnoughPeers bool) {
```

Comment:
so we can just assume now that if not enough peers, there was some sort of issue with disc

Reply:
We know that FindPeers can end earlier than the timeout in success case so no